var/spack/repos/builtin/packages/raja/package.py

```diff
@@ -438,17 +438,13 @@ def _test_examples(self):
             ("wave-eqn", [r"Max Error = 2", r"Evolved solution to time"]),
         ]
         for exe, expected in checks:
-            reason = "test: checking output of {0} for {1}".format(exe, expected)
-            self.run_test(
-                exe,
-                [],
-                expected,
-                installed=False,
-                purpose=reason,
-                skip_missing=True,
```

Comment:
This option indicates you need to be skipping the test if the executable is missing.